drivers/hmis/spec/requests/hmis/update_relationship_to_ho_h_spec.rb

```diff
@@ -175,7 +175,7 @@ def perform_mutation
       errors = result.dig('data', 'updateRelationshipToHoH', 'errors')
       expect(enrollment).to be_present
       expect(errors).to be_empty
-      expect(e1.reload.relationship_to_hoh).to eq(99)
+      expect(e1.reload.relationship_to_hoh).to eq(6)
```

Comment:
This should be a 5 as well, I think.
```suggestion
      expect(e1.reload.relationship_to_hoh).to eq(5)
```